README.md

```diff
@@ -1,3 +1,166 @@
+[tuned](https://github.com/redhat-performance/tuned/)
+daemon for [OpenShift](https://openshift.io/) as
+a Kubernetes DaemonSet.  It ensures [custom tuning
+specification](#custom-tuning-specification) is passed to all
+[containerized tuned](https://github.com/openshift/openshift-tuned)
+daemons running in the cluster in the format they understand.
+The daemons run on all nodes in the cluster, one per node.
+
+
+## Deploying the node tuning operator
+
+The operator is deployed by applying the manifests in the operator's
+`/manifests` directory.  It tries to be useful out of the box by
+creating a default deployment and configuration
+([CR](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/))
+for the tuned daemons.  The following shows the default CR created
+by the operator on a cluster with the operator installed.
+
+```
+$ oc get Tuned
+NAME      AGE
+default   1h
+```
+
+## Custom tuning specification
+
+For an example of a tuning specification, please refer to
+`/assets/tuned/07-cr-tuned.yaml` in the operator's directory or to
+the resource created in a live cluster by:
+
+```
+oc get Tuned/default -o yaml
+```
+
+The custom resource for the operator has two major sections.  The first
+section `profile:` is a list of tuned profiles and their names, the second
+`recommend:` defines the profile selection logic.
+
+Multiple custom tuning specifications can co-exist as multiple CRs
+in the operator's namespace.  The existence of new or deletion of
+old CRs is detected by the operator and all existing custom tuning
+specifications are merged and appropriate objects for the containerized
+tuned daemons updated.
+
+### Profile data
+
+The `profile:` section lists tuned profiles and their names.
+
+```
+  profile:
+  - name: tuned_profile_1
+    data: |
+      # Tuned profile specification
+      [main]
+      summary=Description of tuned_profile_1 profile
+
+      [sysctl]
+      net.ipv4.ip_forward=1
+      # ... other sysctl's or other tuned daemon plugins supported by the containerized tuned
+
+  # ...
+
+  - name: tuned_profile_n
+    data: |
+      # Tuned profile specification
+      [main]
+      summary=Description of tuned_profile_n profile
+
+      # tuned_profile_n profile settings
+```
+
+
+### Recommended profiles
+
+The profile selection logic is defined by the `recommend:` section of the CR.
+
+```
+  recommend:
+  - match:                              # optional; if omitted, profile match is assumed unless a profile with a higher matches first
+    <match>                             # an optional array
+    priority: <priority>                # profile ordering priority, lower numbers mean higher priority (0 is the highest priority)
+    profile: <tuned_profile_name>       # e.g. tuned_profile_1
+
+  # ...
+
+  - match:
+    <match>
+    priority: <priority>
+    profile: <tuned_profile_name>       # e.g. tuned_profile_n
+```
+
+If `<match>` is omitted, a profile match (i.e. _true_) is assumed.
```

Comment:
s(/i.e. _true_)/for example, _true_)

Reply:
Are you sure about this change?  This would the semantics.  If `<match>` is omitted then it automatically "matches", i.e. the match evaluates to "true".  "for example" sounds strange here to me.

Reply:
@jmencak Oh okay, I see. We tend to avoid i.e. and e.g. in our docs, but if this statement is more correct as-is, then we can let it be.